### ISSUE TITLE
v2.4.0 RC1: settings menu

### ISSUE THREAD

**Issue by jakobroehrl** [nextcloud/calendar#3640]
![image](https://user-images.githubusercontent.com/24530680/141436350-da281861-dbeb-4851-b4dc-37db40d742a9.png)

v2.3.4:
![image](https://user-images.githubusercontent.com/24530680/141436393-9fca5120-4f87-4b6a-8b4a-cb1aa9abda1f.png)

**Comment by ChristophWurst** [comment]
@GretaD could you debug this?

**Comment by szaimen** [comment]
This works again by using the following rule:
```
#app-settings__content .action-checkbox__label {
display: unset !important;
}
```

**Comment by jakobroehrl** [comment]
solved in the 2.4.0 
Thanks a lot, guys.
